test/distributed/test_aten_comm_compute_reordering.py

```diff
@@ -730,66 +730,6 @@ def func(a, b, c, d, *, ranks):
             correct = func(a, b, c, d, ranks=ranks)
             self.assertTrue(same(out, correct))
 
-    @unittest.skipIf(not HAS_GPU, "Inductor+gpu needs triton and recent GPU arch")
-    @torch._inductor.config.patch(get_bucket_patches(2.0))
```

Comment:
note: this test is a dupe with `test_bucketing_split_for_overlap` not sure why it is even here..